src/ngraph/pass/reshape_sinking.cpp

```diff
@@ -166,6 +166,18 @@ void swim(Input<Node> input, shared_ptr<op::Reshape> reshape)
         auto csw = work_queue.front();
         work_queue.pop_front();
         auto n = csw.input.get_source_output().get_node_shared_ptr();
+        auto materialize = [csw, n]() {
```

Comment:
Thanks for bringing this bug fix to master @ashokei, I thought we had previously but guess not!